src/app/components/navigation/navigation.component.ts

```diff
@@ -41,10 +40,30 @@ export class NavigationComponent implements OnInit {
       .subscribe((state: BreakpointState) => {
         this.hideNav = state.matches;
       });
-    if (localStorage.getItem('bearerToken')) {
-      this.bearerTokenSet = true;
-      this.config.rootUrl = 'https://platform.planqk.de/qc-catalog';
-    }
+    console.log('navigation component ngOnInit');
+    this.planqkPlatformLoginService
+      .isLoggedIn()
+      .subscribe((loggedIn: boolean) => {
+        console.log(loggedIn ? 'logged in' : 'not logged in');
+        if (loggedIn) {
+          this.planqkPlatformLoginService
+            .getBearerToken()
+            .subscribe((token: string) =>
+              console.log('bearer token: ' + token)
+            );
+          console.log(
+            'refresh token: ' +
+              this.planqkPlatformLoginService.getRefreshToken()
+          );
+
+          this.bearerTokenSet = true;
+          this.config.rootUrl = 'https://platform.planqk.de/qc-catalog';
+          this.reloadStartPage();
+          this.utilService.callSnackBar('Successfully logged in.');
+        } else {
+          this.utilService.callSnackBar('Not logged in.');
```

Comment:
you should also mention where you are not logged in. Otherwise it is confusing when opening the UI for the first time

Reply:
done